python/mxnet/contrib/__init__.py

```diff
@@ -28,3 +28,4 @@
 from . import tensorboard
 
 from . import text
+from . import serde
```

Comment:
https://github.com/dmlc/

Reply:
Will change the module name from serde to onnx.

Reply:
+1
Serde can come in later as a facade on top of onnx and coreml